packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -55,48 +55,74 @@ export default class Meetings extends StatelessSparkPlugin {
    * In current nomenclature, this would create a `call`.
    * `spaceID` is used in {https://github.com/webex/spark-js-sdk/blob/master/packages/node_modules/@ciscospark/plugin-phone/src/call.js#L1007 `call.dial`}
    * @param {string} destination - {https://www.cisco.com/c/en/us/td/docs/ios-xml/ios/voice/cube/configuration/cube-book/sip-supp-uri-based-dialing.html sipURL} or spaceID, AKA {https://developer.webex.com/endpoint-rooms-roomId-get.html roomID}
-   * @returns {Meeting} A new Meeting.
+   * @returns {Promise} A new Meeting.
    */
   create(destination) {
     return this.meetingInfo
       .fetchMeetingInfo(destination)
-      .then(() => {
-        // handle response
-      })
-      .catch(() => {}) // handle errors
-      .then(() => {
-        const meeting = new Meeting(
-          {
-            sipUri: destination,
-            userId: this.spark.internal.device.userId,
-            deviceUrl: this.spark.internal.device.url,
-            roapSeq: 0,
-            meetingInfo: {}
-          },
-          {
-            parent: this.spark
-          }
-        );
+      .then((info) => info)
+      .catch((err) => err)
+      .then((info) => this.createMeeting(info, destination))
+      .catch((err) => err);
+  }
 
-        return MeetingCollection.set(meeting);
```

Comment:
Do we no longer need to set it in the collection?

Reply:
it got moved to this.createMeeting